epregressions/diffs/math_diff.py

```diff
@@ -182,7 +182,11 @@ def dict_of_dicts2dict_of_lists(dict_of_dicts, key_order, list_labels):
     for key in key_order:
         dict_of_lists[key] = []
         for ll in list_labels:
-            dict_of_lists[key].append(dict_of_dicts[key][ll])
+            try:
+                dict_of_lists[key].append(dict_of_dicts[key][ll])
```

Comment:
This condition can raise an exception when the numerics are weird, like `1.0.0`.  Catching them here just helps clear up the weird issue that would otherwise occur.